frontend/src/authschemes/webauthn/settings/index.tsx

```diff
@@ -57,18 +58,22 @@ const CredentialList = (props: {
                 <tr key={credentialName}>
                   <td>{credentialName}</td>
                   <td>{toEnUSDate(dateCreated)}</td>
-                  <td>
-                    <Button small danger onClick={() => {
-                      deleteModal.show({ credentialName })
-                    }}>
-                      Delete
-                    </Button>
+                  <td className={cx('button-cell')}>
+                    <ButtonGroup className={cx('row-buttons')}>
+                      <Button small onClick={() => {
+                        modifyModal.show({ credentialName })
+                      }}>Edit</Button>
+                      <Button danger small onClick={() => {
+                        deleteModal.show({ credentialName })
+                      }}>Delete</Button>
+                    </ButtonGroup>
                   </td>
                 </tr>
               )
             })}
           </Table>
           {renderModals(deleteModal)}
+          {renderModals(modifyModal)}
```

Comment:
So, I'm just realizing that renderModals can take any number of modals (even though we don't use it this way). it might be worth looking into to see if we can remove the extra renderModal calls.

Reply:
ahh nice

Reply:
fixed